### PR TITLE
close #136 by preventing scrolling while modal window is open

### LINKED ISSUE #136: make modal window prevent background scrolling while active
This issue helps us keep track of making the background static, i.e., not scrollable, while the modal window is open.

### PR DESCRIPTION
**Pull request by hamirmahal**
This pull request will close #136 if we merge it by preventing the background from being scrollable while the redirection notification modal window is open.

It was a really simple, few lines of code implementation like you said.

```
.ReactModal__Body--open {
  overflow: hidden;
}
```

I found that in the [classes section of the documentation for react-modal](http://reactcommunity.org/react-modal/styles/classes/). Hope it helps!

### CODE REVIEW

File: src/tailwind.css [hunk 1]
```diff
@@ -1805,18 +1805,6 @@ video {
   border-radius: 0.5rem;
 }
 
-.rounded-xl {
```
Comment (amilajack) [inline src/tailwind.css:1808]: Did you run `build:style` after updating styles.css? Not sure why these generated classes were removed

**Comment by amilajack** [comment]
Nice! 💯 Super excited to get this in

File: src/tailwind.css (hunk 1 above)
Comment (hamirmahal) [reply src/tailwind.css:1808]: I did. We have to run `build:style` to make CSS changes show up, right?

**Review by amilajack — APPROVED** [review]
🥳 LGTM!

File: src/tailwind.css (hunk 1 above)
Comment (amilajack) [reply src/tailwind.css:1808]: took a look at the netlify, everything looks good!

**Pull request merged**